x-pack/metricbeat/module/azure/_meta/kibana/7/dashboard/Metricbeat-azure-database-account-overview.json

```diff
@@ -0,0 +1,715 @@
+          "aggs": [],
+          "params": {
+            "controls": [
+              {
+                "fieldName": "azure.subscription_id",
+                "id": "1584710440054",
+                "indexPatternRefName": "control_0_index_pattern",
+                "label": "Subscription",
+                "options": {
+                  "dynamicOptions": true,
+                  "multiselect": false,
+                  "order": "desc",
+                  "size": 5,
+                  "type": "terms"
+                },
+                "parent": "",
+                "type": "list"
+              },
+              {
+                "fieldName": "azure.resource.group",
+                "id": "1584710497045",
+                "indexPatternRefName": "control_1_index_pattern",
+                "label": "Resource Group",
+                "options": {
+                  "dynamicOptions": true,
+                  "multiselect": false,
+                  "order": "desc",
+                  "size": 5,
+                  "type": "terms"
+                },
+                "parent": "",
+                "type": "list"
+              },
+              {
+                "fieldName": "cloud.instance.name",
+                "id": "1584710535722",
+                "indexPatternRefName": "control_2_index_pattern",
+                "label": "Resource",
+                "options": {
+                  "dynamicOptions": true,
+                  "multiselect": false,
+                  "order": "desc",
+                  "size": 5,
+                  "type": "terms"
+                },
+                "parent": "",
+                "type": "list"
+              }
+            ],
+            "pinFilters": false,
+            "updateFiltersOnChange": true,
+            "useTimeFilter": false
+          },
+          "title": "Database Account Filters [Metricbeat Azure]",
+          "type": "input_control_vis"
+        }
+      },
+      "id": "674c1d70-83cc-11ea-be84-f5d4d6b9a792",
+      "migrationVersion": {
+        "visualization": "7.4.2"
+      },
+      "references": [
+        {
+          "id": "metricbeat-*",
+          "name": "control_0_index_pattern",
+          "type": "index-pattern"
+        },
+        {
+          "id": "metricbeat-*",
+          "name": "control_1_index_pattern",
+          "type": "index-pattern"
+        },
+        {
+          "id": "metricbeat-*",
+          "name": "control_2_index_pattern",
+          "type": "index-pattern"
+        }
+      ],
+      "type": "visualization",
+      "updated_at": "2020-04-21T12:34:11.783Z",
+      "version": "WzY5MDEsMTFd"
+    },
+    {
+      "attributes": {
+        "description": "",
+        "kibanaSavedObjectMeta": {
+          "searchSourceJSON": {
+            "filter": [],
+            "query": {
+              "language": "kuery",
+              "query": ""
+            }
+          }
+        },
+        "title": "Database Account Total Requests [Metricbeat Azure]",
```

Comment:
I think total request can be further split to total requests by `StatusCode` or `Status` (whichever makes more sense)